arviz/rcparams.py

```diff
@@ -93,11 +102,64 @@ def _validate_boolean(value):
     return value is True or value == "true"
 
 
+def iterable_validator(validate_scalar, length=None):
+    """Validate value is an iterable datatype."""
+    # no blank lines allowed after function docstring by pydocstyle,
+    # but black requires white line before function
```

Comment:
I would use the comments to give attribution to matplotlib's listify validator

Reply:
Makes sense. Thanks!